### PR TITLE
Fix 404 and refactor like `testcase.apply(TestcaseIoPatching)`

### PR DESCRIPTION
**Pull request by Johnny850807**
Fix 404 and refactor like `testcase.apply(TestcaseIoPatching)`

### CODE REVIEW

File: Commons/src/main/java/tw/waterball/judgegirl/commons/utils/CollectionUtils.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+/**
+ * @author - johnny850807@gmail.com (Waterball)
+ */
+public class CollectionUtils {
+    @SafeVarargs
+    public static <T> Collection<T> merge(Collection<T>... collection) {
+        return stream(collection)
+                .flatMap(Collection::stream)
+                .collect(toList());
+    }
```
Comment (Wally5077) [inline Commons/src/main/java/tw/waterball/judgegirl/commons/utils/CollectionUtils.java:17]:
```Java
@SafeVarargs
    public static <T> Collection<T> merge(Collection<T>... collections) {
        return flapMapToList(collections, Collection::stream);
    }
```

**Review by Wally5077 — CHANGES_REQUESTED** [review]
Review done

File: Commons/src/main/java/tw/waterball/judgegirl/commons/utils/CollectionUtils.java (hunk 1 above)
Comment (Johnny850807) [reply Commons/src/main/java/tw/waterball/judgegirl/commons/utils/CollectionUtils.java]: I delete this class

**Review by Wally5077 — APPROVED** [review]
LGTM

**Pull request merged**